lambdas/credentialissuerstart/src/main/java/uk/gov/di/ipv/core/credentialissuer/CredentialIssuerStartHandler.java

```diff
@@ -184,4 +215,13 @@ private String getIpvSessionId(Map<String, String> headers)
         }
         return ipvSessionId;
     }
+
+    @Tracing
+    private void persistOauthState(String ipvSessionId, String criId, UUID oauthState) {
+        IpvSessionItem ipvSessionItem = ipvSessionService.getIpvSession(ipvSessionId);
+        CredentialIssuerSessionDetailsDto credentialIssuerSessionDetailsDto =
+                new CredentialIssuerSessionDetailsDto(criId, oauthState.toString());
+        ipvSessionItem.setCredentialIssuerSessionDetails(credentialIssuerSessionDetailsDto);
+        ipvSessionService.updateIpvSession(ipvSessionItem);
```

Comment:
The cri start lambda will need some new permissions to be able to read/write into the dynamoDB table. It will also need the IPV_SESSIONS_TABLE_NAME env variable

Reply:
Done!